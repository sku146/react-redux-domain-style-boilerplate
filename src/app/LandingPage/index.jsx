import React, { Component } from 'react';
import PropTypes from 'prop-types';

class LandingPage extends Component {
  render() {
    return (
      <div>React Redux Boilerplate</div>
    )
  }
}

export default LandingPage;
