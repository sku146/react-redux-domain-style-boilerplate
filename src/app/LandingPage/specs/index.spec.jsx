import React from 'react';
import LandingPage from '../index';

describe('LandingPage', () => {
  const defaultProps = {};

  const RendererComponent = props => renderer.create(<LandingPage {...props} />);

  describe('@render', () => {
    it('should render as expected', () => {
      const component = RendererComponent(defaultProps);
      expect(component.toJSON()).toMatchSnapshot();
    });
  });
});
