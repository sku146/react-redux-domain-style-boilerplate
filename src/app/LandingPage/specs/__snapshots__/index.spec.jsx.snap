// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage @render should render as expected 1`] = `
<div>
  React Redux Boilerplate
</div>
`;
